frontend/sw360-portlet/src/main/java/com/siemens/sw360/portal/portlets/projects/ProjectPortlet.java

```diff
@@ -449,7 +464,7 @@ private void prepareStandardView(RenderRequest request) throws IOException {
             } else {
                 projectList = projectClient.refineSearch(searchtext, filterMap, user);
             }
-            for(Project project:projectList){
+            for (Project project : projectList) {
```

Comment:
Sorry, it's my IDE rather than my love for spaces ;-). I should be more economic with automatic alignment of entire files.